qa/suites/fs/workload/replication/default.yaml

```diff
@@ -0,0 +1 @@
+# Use default (8000)
```

Comment:
default.yaml is not _really_ needed, isn't it?

... or was it the reason metadata was not replicated as expected? (that would be weird).

Reply:
ah ok.

I assumed that we _always_ want to exercise metadata replication for tests.